### ISSUE TITLE
Friendly note about accessibility improvements

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#49]
Hi team! Just wanted to kindly point out that accessibility features seem to be missing in the current version of the study group map. It would be great to make the platform more inclusive and accessible to everyone. Thanks for considering this! Keep up the great work!
